README.md

```diff
@@ -84,11 +77,46 @@ Internally it uses [untyped](https://untyped.unjs.io/) and [jiti](https://github
 
     <!-- AUTOMD_END -->
 
-### Args supported for `jsdocs`
+#### Args supported for `jsdocs`
 
 - `src`: Path to the source file. The default is `./src/index` and can be omitted.
 - `headingLevel`: Nested level for markdown group headings (default is `2` => `##`). Note: Each function uses `headingLevel+1` for the title in nested levels.
-- `group`: Only render function exportes annotated with `@group name`. By default, there is no group filter. Value can be a string or array of strings.
+- `group`: Only render function exports annotated with `@group name`. By default, there is no group filter. Value can be a string or array of strings.
+
+### `pm-install` Generator
+
+`pm-install` Generator generates commands for several JavaScript package managers.
+
+#### Usage
+
+    <!-- AUTOMD_START generator="pm-install" name="package-name" dev="true" -->
+    <!-- AUTOMD_END -->
+
+**Updated Result:**
+
+    <!-- AUTOMD_START generator="pm-install" name="package-name" dev="true" -->
+
+    ```sh
+    # npm
+    npm install package-name -D
+
+    # yarn
+    yarn add package-name -D
+
+    # pnpm
+    pnpm install package-name -D
+
+    # bun
+    bun install package-name -D
+    ```
+
+    <!-- AUTOMD_END -->
+
+#### Args supported for `pm-install`
```

Comment:
```suggestion
#### Arguments
```

Reply:
I will leave this up to @pi0 since he already changed these.

Reply:
Sounds good, I think since these are already headings underneath the "`xyz` Generator" section headings we don't need to repeat "supported for `pm-install`"...

Reply:
Makes sense